README.md

```diff
@@ -0,0 +1,147 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+  <h1><b> Ulises Cappa - Microverse Student</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Future Features](#future-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Test](#test)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Awesome Books App <a name="about-project"></a>
+
+**Awesome Books App** It's an application where users will Add or Remove books that are acknowledged as good to our personal library.
+
+In this project we mostly used HTML,JavaScript & a bit of CSS skills to complete the project.
+
+As well as knowing how to manipulate information but, more importantly, how to copy a design using a medium-wire frame.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Great Design and Colors.**
+- **Good Practices within each File.**
+- **Excellent use of Local Storage.**
+- **Code divided into modules**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Future Features <a name="future-features"></a>
+
+- **Implement a CSS Preprocessor.**
+- **Implement a Database**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+### Take a Look
+
+- [Live Demo Link](https://godedict.github.io/Awesome-books/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git@github.com:ulises2607/Awesome-Books-v2.git
+cd Awesome-Project-v2
+code . (To open VS Code Editor)
+```
+
```

Comment:
- you have omitted the `Usage section` which is to describe how the project can be viewed in the browser by whoever chooses to view it locally. I kindly suggest you make this addititon to your readme 👍 ✨